app/models/concerns/symphony_request.rb

```diff
@@ -6,7 +6,7 @@ def appears_in_myaccount?
   end
 
   def send_to_symphony!(options = {})
-    SubmitSymphonyRequestJob.perform_now(self, options)
+    SubmitSymphonyRequestJob.perform_later(id, options)
```

Comment:
I was thinking we could pass something into this method that would allow us to switch on `perform_now` vs `perform_later` (so that mediated item approval can be run inline).   More than happy to take a look at that with you if you would like.